spring-modulith-junit/src/main/java/org/springframework/modulith/junit/TestExecutionCondition.java

```diff
@@ -59,7 +59,12 @@ ConditionEvaluationResult evaluate(ConditionContext context) {
 		}
 
 		if (!changes.hasClassChanges()) {
-			return enabled("No source file changes detected.");
+			var environment = new StandardEnvironment();
+			ConfigDataEnvironmentPostProcessor.applyTo(environment);
+
+			return "execute-none".equals(environment.getProperty("spring.modulith.test.on-no-changes"))
```

Comment:
a switch expression could be used here. ("execute-none", "execute-all")
but i would actually prefer a boolean but a renaming of "spring.modulith.test.on-no-changes" would then make sense like "spring.modulith.test.skip-all-on-no-class-changes"

Reply:
enum sounds great. i just dont know if i understand it correctly.
i pushed an attempt.